package.json

```diff
@@ -69,6 +69,7 @@
   "dependencies": {
     "inversify": "^6.0.1",
     "inversify-binding-decorators": "^4.0.0",
+    "multer": "^1.4.5-lts.1",
```

Comment:
Remove the dependency,, it's not necessary. You have to copy the interfaces from multer.
As far as I know, the interface is under:
```typescript
declare Multer {
   interface Options {
   }
}
```